tests/test_cli.py

```diff
@@ -126,10 +126,21 @@ def test_that_lyrics_does_not_break_with_request_giving_wrong_status_code(self,
 	@patch('swaglyrics.cli.get_lyrics', return_value=None)
 	def test_that_lyrics_calls_requests(self, f_get_lyrics, mock_requests):
 		"""
-		Test that get_lyrics calls requests
+		Test that get_lyrics calls requests when not a trivial case 
 		"""
 		self.assertEqual(lyrics(
-			"Pixel2XL", "Elgoog", True), "Couldn't get lyrics for Pixel2XL by Elgoog.\nPhone is dope")
+			"Pixel2XL!", "Elgoog", True), "Couldn't get lyrics for Pixel2XL! by Elgoog.\nPhone is dope")
+		self.assertEqual(lyrics(
+			"Pixel2XL", "Elgoog", True), "Couldn't get lyrics for Pixel2XL by Elgoog.\n")
+
+	@patch('swaglyrics.cli.requests.post', return_value=R(404))
+	@patch('swaglyrics.cli.get_lyrics', return_value=None)
+	def test_that_lyrics_requests_canfail(self, f_get_lyrics, mock_requests):
+		"""
+		Test that get_lyrics when not a trivial case calls requests
```

Comment:
I don't think this comment is accurate because a non trivial case is already there in the previous test

Reply:
Ok, I fixed it